refactor(curriculum): restructure sudoku solver python project (#54303)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/07-scientific-computing-with-python/learn-classes-and-objects-by-building-a-sudoku-solver/6606b8d31356fe4563f0e99c.md

---
id: 6606b8d31356fe4563f0e99c
title: Step 20
challengeType: 20
dashedName: step-20
---

# --description--

> Outside the `for` loop, return `None`. This addresses the case in which no empty cell is found and the sudoku board is completely filled.

Reviewer: ```suggestion
Outside the `for` loop, return `None`. This handles the case in which no empty cell is found, indicating that the sudoku board is completely filled.
```
